src/api/app/models/bs_request_action_change_devel.rb

```diff
@@ -24,7 +24,7 @@ def execute_accept(_opts)
     target_package.develpackage = Package.get_by_project_and_name(self.source_project, self.source_package)
 
     target_package.resolve_devel_package
-    target_package.store(comment: "change_devel request #{self.bs_request.id}", requestid: self.bs_request.id)
+    target_package.store(comment: "change_devel request #{self.bs_request.number}", requestid: self.bs_request.number)
```

Comment:
It is intended, since requestid here ist the identifier in the backend.

But I really need to add a comment for this here... or maybe even better just handover the request and let the package model do the right thing(tm)...